.github/workflows/ci.yml

```diff
@@ -457,11 +479,67 @@ jobs:
           sudo apt-get install clang
           ## sudo apt-get install --quiet -y libsdl2-dev libsdl2-ttf-dev libsdl2-mixer-dev libsdl2-image-dev
       - name: Build V
-        run: make -j4 && ./v -cc clang -cg -cflags -Werror -o v cmd/v
+        run: make -j4 && ./v -cg -cflags -Werror -o v cmd/v
+      - name: Self tests (-fsanitize=address)
+        run: ASAN_OPTIONS=detect_leaks=0 ./v -cflags -fsanitize=address test-self
+      - name: Self tests (-asan-compiler)
+        run: 
+          ./v -cflags -fsanitize=address -o v cmd/v &&
+          ASAN_OPTIONS=detect_leaks=0 ./v -cc tcc test-self -asan-compiler
```

Comment:
should not this be using `-cc clang` ?

Reply:
No, here we are testing if the compiler can compile the programs/tests without address errors.  The programs themselves do not matter, because they are sanitized in the task above.

Reply:
Thanks for the explanation. That is subtle 🤔 . 
Can you rename the task to something like `V compiled with -fsanitize=address works` ?

Reply:
Sure, good idea :slightly_smiling_face: